src/main/java/Excavator.java

```diff
@@ -0,0 +1,9 @@
+class Excavator extends Machine {
+    public void doWork() {
+        System.out.println("Excavator started working.");
+    }
+
+    public void stopWork() {
+        System.out.println("Excavator stopped working.");
```

Comment:
The message in `stopWork` method should also be more informative. As with `doWork`, include the class name dynamically.